keyboards/session.php

```diff
@@ -31,6 +31,10 @@
   $embed_android = $embed == 'android';
   $embed_ios = $embed == 'ios';
 
+  if(isset($_REQUEST['locale'])) {
+    \Keyman\Site\com\keyman\Locale::overrideCurrentLocale($_REQUEST['locale']);
+  }
```

Comment:
OK, but we do need to validate that it's an acceptable locale

Reply:
TODO on follow-on PR once I add some .po files and workout how it fallsback to en strings.